Observer/CopyFieldsFromQuoteToOrderObserver.php

```diff
@@ -35,6 +35,11 @@ public function execute(\Magento\Framework\Event\Observer $observer)
         /* @var \Magento\Quote\Model\Quote $quote */
         $quote = $observer->getEvent()->getData('quote');
 
+        if ($cartExtensions = $quote->getExtensionAttributes()) {
```

Comment:
`=` or `==` ?

Reply:
It should be `=`. I have moved the `$cartExtensions` declaration outside of the `if` statement